src/main/webapp/scripts/my-trips.js

```diff
@@ -198,16 +198,19 @@ async function findHotel() {
  */
 async function parseAndRenderHotelResults(json, centerPoint) {
   const modalContent = document.getElementById("hotel-results");
-  if (!json) {
-    modalContent.innerText = "No hotels nearby. Sorry.";
+  const hotelsMapElem = document.getElementById("hotels-map");
+  hotelsMapElem.style.height = "";
+  hotelsMapElem.innerHTML = "";
+  if (!json || json.length === 0) {
+    modalContent.innerText =
+      "We couldn't find any hotels nearby. Sorry about that.";
   } else {
-    const hotelsMapElem = document.getElementById("hotels-map");
     json = json.slice(0, 10);
     const hotelMap = new google.maps.Map(
       document.getElementById("hotels-map"),
       {
         center: centerPoint,
-        zoom: 12
+        zoom: 12,
```

Comment:
nit: Is the extra comma for future map options?

Reply:
I think Prettier recommends this for good style. I like trailing commas for object literals since it makes adding properties easier in the future.